docs/index.rst

```diff
@@ -16,6 +16,12 @@ Developer Documentation
     (Legacy) Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>
     developers_guide/micro_frontends_in_open_edx
 
+Getting Started
+###############
+In a hurry to start writing code? Go setup our `Devstack`_
```

Comment:
```suggestion
In a hurry to start writing code? Go set up our `Devstack`_.
```
I think this could use a little more guidance.
```suggestion
In a hurry to start writing code? Go set up `Devstack`_, our local Docker-based development environment.
```